packages/fxa-payments-server/src/components/LoadingOverlay/index.tsx

```diff
@@ -11,7 +10,11 @@ type LoadingOverlayProps = {
 
 export const LoadingOverlay = ({ isLoading }: LoadingOverlayProps) =>
   !isLoading ? null : (
-    <div id="loading-overlay" data-testid="loading-overlay">
-      <img alt="" src={spinnerImage} />
+    <div
+      id="loading-overlay"
```

Comment:
Can we rip `id="loading-overlay"` out now?

Reply:
Hahaha yes! My bad. Forgot to remove it after deleting the stylesheet. 🙂 Thank you!